backtester/eventhandlers/exchange/exchange.go

```diff
@@ -103,19 +103,18 @@ func (e *Exchange) ExecuteOrder(o order.Event, data data.Handler, bot *engine.En
 	} else {
 		limitReducedAmount = reducedAmount
 	}
-
 	// Conforms the amount to fall into the minimum size and maximum size limit after reduced
 	switch f.GetDirection() {
 	case gctorder.Buy:
-		if limitReducedAmount < cs.BuySide.MinimumSize || limitReducedAmount > cs.BuySide.MaximumSize {
+		if (limitReducedAmount < cs.BuySide.MinimumSize || limitReducedAmount > cs.BuySide.MaximumSize) && (cs.BuySide.MinimumSize > 0 && cs.BuySide.MaximumSize > 0) {
```

Comment:
Thanks for that change! Last last thing! 😄 
```
if (limitReducedAmount < cs.BuySide.MinimumSize && cs.BuySide.MinimumSize > 0) || (limitReducedAmount > cs.BuySide.MaximumSize && cs.BuySide.MaximumSize > 0) {
```
Same for SellSide. Just in the rarer event that someone has a min OR max limit set.

Reply:
what about a check "cs.BuySide.MaximumSize > 0" ? It doesn't matter if MinimumSize is zero or not. BUT MaximumSize Does care.
(limitReducedAmount < cs.BuySide.MinimumSize || limitReducedAmount > cs.BuySide.MaximumSize) && cs.BuySide.MaximumSize > 0

Reply:
how it works this time.